common/validations.js

```diff
@@ -0,0 +1,58 @@
+const CONSTANT = require('./constant')
+module.exports = {
+
+    LOGIN:(userData, cb)=>{
+        if(userData.email == undefined || userData.email == '')
+            cb( CONSTANT.EMAIL_BLANK_ERROR, null)
+        else if(userData.password == undefined || userData.password == '')
+            cb( CONSTANT.PASSWORD_BLANK_ERROR, null)
+        else
+            cb(null, "OK")
+    },
+
+    SEND_OTP:(email, cb)=>{
+        if(email == undefined || email == '')
+            cb( CONSTANT.EMAIL_BLANK_ERROR, null )
+        else
+            cb(null, "OK")
+    },
+
+    VERIFY_OTP:(userData, cb)=>{
+        if(userData.email == undefined || userData.email == '')
```

Comment:
this line should be like this `if(userData.email == undefined || userData.email == '')` -> `if(userData.email)`